parsers/svgo-v3/README.md

```diff
@@ -0,0 +1,185 @@
+# SVGO
+
+## Description
+
+This parser helps you optimize vectors using [svgo](https://github.com/svg/svgo).
```

Comment:
This parser helps you optimize vectors using [svgo](https://github.com/svg/svgo) v3.